volumina/utility/signalingDefaultDict.py

```diff
@@ -29,13 +29,13 @@ def __getitem__(self, key):        return self._dict[key]
     def __iter__(self):                return self._dict.__iter__()
     def __contains__(self, key):       return self._dict.__contains__(key)
     def get(self, key, default=None):  return self._dict.get(key, default)
-    def viewkeys(self):                return self._dict.viewkeys()
-    def items(self):                   return self._dict.items()
-    def iteritems(self):               return self._dict.iteritems()
-    def keys(self):                    return self._dict.keys()
-    def iterkeys(self):                return self._dict.iterkeys()
-    def itervalues(self):              return self._dict.itervalues()
-    def values(self):                  return self._dict.values()
+    def viewkeys(self):                return self._dict.keys()
+    def items(self):                   return list(self._dict.items())
+    def iteritems(self):               return iter(self._dict.items())
+    def keys(self):                    return list(self._dict.keys())
+    def iterkeys(self):                return iter(self._dict.keys())
+    def itervalues(self):              return iter(self._dict.values())
+    def values(self):                  return list(self._dict.values())
```

Comment:
The idea with this class is to mimic the `dict` API, so `keys()`, `values()`, `items()` should return iterators.  And we should delete `iterkeys()`, etc.